website/docs/guides/migration/from-v2-to-v3.md

```diff
@@ -244,6 +244,9 @@ For the KafkaFlow engine to adapt to the dynamic change of workers, the way the
 
 To achieve this, a new method in the message context was provided to signal the message completion `.Complete()`. Additionally, a new property called `AutoMessageCompletion` was added to signal if KafkaFlow should signal the message as completed automatically which is the default behavior. This property is useful for scenarios where we don't want to complete the message right away, for example in Batch Consume scenarios.
 
+- Use `WithManualMessageCompletion()` instead of `WithManualStoreOffsets`.
+- Use `context.ConsumerContext.Complete()` instead of `context.ConsumerContext.StoreOffset();`.
```

Comment:
```suggestion
- Use `WithManualMessageCompletion()` instead of `WithManualStoreOffsets()`.
- Use `context.ConsumerContext.Complete()` instead of `context.ConsumerContext.StoreOffset()`.
```

Reply:
Fixed